src/HomePage/components/Search/Search.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import "./SearchStyles.css"
+
+export function Search(props: { onSearchMovie: (arg: string) => void; }) {
+    const handleChange = (e: any) => {
```

Comment:
Please avoid using any, we have bunch of React event types ready to be used for this